master/internal/task/allocation.go

```diff
@@ -173,10 +173,18 @@ func (a *Allocation) Receive(ctx *actor.Context) error {
 		a.Cleanup(ctx)
 	case sproto.ContainerLog:
 		if a.req.StreamEvents != nil {
-			ctx.Tell(a.req.StreamEvents.To, sproto.Event{LogEvent: ptrs.StringPtr(msg.String())})
+			ctx.Tell(a.req.StreamEvents.To, sproto.Event{
```

Comment:
For every struct initialization as reported by my editor, I just added the previously present fields.